Create PM-Meeting-notes-24Jan24.md (#3507)
the-turing-way/the-turing-way: governance/pm-core-meeting-notes/24Jan24.md

# The Turing Way - Core Project Management Meetings 24 January

## Bi-weekly meetings (with some exceptions)

**Participants** 
This call is joined by *The Turing Way* Core Staff:

* Kirstie Whitaker
* Malvika Sharan 
* Alexandra Araujo Alvarez 
* Arielle Bennett 

## Agenda

Review together the agenda and agree on priorities and timings.  

:hourglass_flowing_sand: **Timing:** Total 90 min

| Duration | Activity |
| ---- | -------- |
| Start | 👋 Welcome |
| 05 mins | Check in   |
| 20 mins | Infrastructure WG   
| 10 mins | TTW Community Call Agenda (15 Feb)
| 20 min | Collaborations Workshop (TPS & TTW Participation) 
| 10 min | CZI Localisation Proposal / Jupyterhub updates
| 20 mins | Other
5 mins | Wrap up  

> 

Author: ## Actions from the meeting:
- [ ] (AAA): update the Domain request with all the proposal - https://github.com/the-turing-way/the-turing-way/issues/3266
- [ ] (MS) New Working Group - Communications WG —> MS suggested to open an Expression of interest to understand and explain what is needed to create this wg, who has capacity to do it? MS will think about it and will bring an ask for a proposal of Expression of interest to the next meeting
- [X] (AAA) Ask the infrastructure wg to open an issue for guidance to maintain a light-weight repo - (https://github.com/the-turing-way/the-turing-way/issues/3516)
- [ ] (ALS/AAA) Start an issue with "Best design of a working group repo, how we want them to be structured, what is needed, easy templates of the minimum they require to facilitate contributions
- [ ] (ALS/AAA) best design to align the community to navigate on the different repos (issue)

## Infrastructure WG

### Buy a URL for The Turing Way 

Documentation here: https://github.com/the-turing-way/the-turing-way/issues/3266

#### Reasons to buy the URL and level of importance:

> - The domain name is giving by netlify if we swap to readthedocs, the URL will find a broken webpage. 

Reviewer: ```suggestion
- The domain name is given by netlify. If we swap to readthedocs, the URL will find a broken webpage. 
```

> - If we buy one, we will redirect trafic from one to another url. While we could redirect from netlify to readthedocs, it is much better to type theturingway.org (or the prefferable domain). We don't need to rely on netlify or readthedocs.

Reviewer: ```suggestion
- If we buy one, we will redirect traffic from one to another url. While we could redirect from netlify to readthedocs, it is much better to type theturingway.org (or the preferable domain). We don't need to rely on netlify or readthedocs.
```

- Proposed planning/design work (developed by Jim):
    - Domain structure:
        - root - the-turing-way.org (redirects to start page)
        - startpage - start.the-turing-way.org (based ond [this](https://the-turing-way.start.page/)?)
        - book - book.the-turing-way.org
        - calendar - cal.the-turing-way.org (who and how would this be maintained / automated?) 
        - etc.
        - SSL certs (if we need to do that ourselves) - Let's Encrypt

#### Pain points:
- Maintainance ownership: (as per Jim contribution) "It is worth thinking about bus factor when we plan how to purchase the domain name and manage DNS. We don't want to have to require a particular person to make any changes or fix problems.I like the idea of the Infrastructure WG being responsible."
- Budget holder / Credit card details: shouldn't this be paid by The Alan Turing? If not, it would be safe to share credentials. Ideas?
    - theturingway.org is £11.01 first year, £19.97 after (on godaddy)
    - £5.92 on namecheap
- Identify and decide the URL name, who and how is this decided?
    - Examples: theturingway.org / theturingway.io 

> Notes and actions: 

Reviewer: ```suggestion
### Notes and actions: 
```

> ### Notes and actions: 

Reviewer 2: @AlexandraAAJ - mixing together notes and actions is a little confusing - do you think you could pull out the specific actions please? The notes are great but it makes scanning through to know exactly what the tasks are a little difficult!

Maybe a summary at the top of the notes?

Author: Thanks Kirstie, I've added the action on top.

MS: 
- Old conversation, there's a strong demand from the community. 
- Huge benefit to have something localise in a space
KW:

> - 2022/23 there was a lot that the community wants this without actual proposal  ---> Encourage working groups to prepare proper request, to have more different pathways 

Reviewer: ```suggestion
- 2022/23 there was a lot of general discussions framed around "the community wants this" without an actual proposal  ---> Encourage working groups to prepare proper request, to have more pathways to make decisions and improve transparent decision making
```

> - The benefit of saying yes are very high, happy to approve the request nut not without a proper request

Reviewer: ```suggestion
- The benefits of saying yes are very high, happy to approve the request but not without a proper request
```

- Simple proposal, remove WIP, add DPAP, and get the WG to approve the official request. Full issue, with all the info and how the challenges are going to be addressed. 
    - Name: suggestions of what is available 
    - Budget: on the RPM / KW credit card details
    - DPAP: (Kit support)

>     - Maintance:

Reviewer: ```suggestion
    - Maintenance:
```

>     - Reputational problem risk: internally at the http://theturingway.org/bookTuring Institute

Reviewer: ```suggestion
    - Reputational problem risk: internally at the Turing Institute
```
